encrypted-media-respec.html

```diff
@@ -1028,7 +1035,7 @@
                   <li><p>Let <var>content type</var> be <var>requested media capability</var>'s <a def-id="capability-contentType"></a> member.</p></li>
                   <li><p>Let <var>robustness</var> be <var>requested media capability</var>'s <a def-id="capability-robustness"></a> member.</p></li>
                   <li><p>If <var>content type</var> is the empty string, return <code>null</code>.</p></li><!-- Invalid input. -->
-                  <li><p>If <var>content type</var> is an invalid or unrecognized MIME type, continue to the next iteration.</p></li>
+                  <li><p>If <var>content type</var> is not a <a def-id="valid-media-mime-type"></a>, continue to the next iteration.</p></li>
```

Comment:
This drops the "... or unrecognized". It is possible for the MIME type to be valid but unrecognized. It may be difficult for a UA to determine whether an unrecognized MIME type is valid or not.